stable/traefik/templates/service.yaml

```diff
@@ -7,6 +7,13 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  {{- range $key, $value := .Values.service.labels }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
```

Comment:
Testing this, I found that you're really going to have to wrap this in a double-if condition similar to what you have done in #414.